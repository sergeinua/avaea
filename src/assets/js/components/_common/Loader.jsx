import React from 'react';

let Loader = React.createClass({

  render: function () {
    return <div className="spinner-holder">
      <div className="icon-spinner"></div>
    </div>
  }
});

export default Loader;
