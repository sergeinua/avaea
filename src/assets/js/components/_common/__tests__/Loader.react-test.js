import React from 'react';
import Loader from '../Loader';
import renderer from 'react-test-renderer';

test('Loader component test', () => {

  var component = renderer.create(
    <Loader />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
