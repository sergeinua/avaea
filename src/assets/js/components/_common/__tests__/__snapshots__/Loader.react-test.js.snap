exports[`test Loader component test 1`] = `
<div
  className="page-loader">
  <div
    className="spinner-holder">
    <div
      className="icon-spinner" />
  </div>
</div>
`;
